docs/_policies/workgroups.md

```diff
@@ -1,29 +1,22 @@
 
-## Adding/Removing Users to your Workgroup
-If you are an Administrator of a workgroup, you can use the Workgroup Manager interface to easily add users to and remove users from that workgroup. Adding users to a workgroup immediately grants them permissions to whatever resource that workgroup controls access to. Likewise, removing users revokes those permissions. You might want to do this if you no longer want someone to have access to your project space or as part of regular auditing of external collaborators you have sponsored access to the Yens for.
+## What is a Workgroup?
 
-Users can be added as Members and/or Administrators of a workgroup. While Members are simply granted permission to the resource controlled by the workgroup, Administrators have the power to add *other* users as Members/Administrators to the workgroup.
+The [Stanford Workgroup Manager](https://workgroup.stanford.edu/){:target="_blank"}, is a web interface that allows Stanford community members to define groups of other community members based on their SUNet IDs for use in various applications. Each workgroup is assigned a unique name that succintly describes its purpose. 
 
-!!! warning
-    Given that Administrators of a workgroup have the power to add other users, please be mindful of who you grant Administrator privileges to. It is *your responsibility* to audit both the Administrators and Members of your workgroup.
+Relevant to the Yens, the Workgroup Manager helps control access to individual [project space](/_policies/services){:target="_blank"} on ZFS (e.g., gsb-rc:[faculty-SUNet]-[projectname]). For example, if your project is stored at `/zfs/projects/faculty/jdoe-housing`, there will be a corresponding workgroup named `gsb-rc:jdoe-housing`.
 
+## How to Add or Remove Users to Your Workgroup?
 
-For instructions on how to perform these operations, use this [helpful page](https://uit.stanford.edu/service/workgroup/add-remove-members) created by Stanford UIT that illustrates these processes with screenshots.
+If you are an Administrator of a workgroup, you can use the Stanford Workgroup Manager to easily add users to and remove users from that workgroup. Adding users to a workgroup immediately grants them permissions to whatever resource that workgroup controls access to. Likewise, removing users revokes those permissions. You might want to do this if you no longer want someone to have access to your project space or as part of regular auditing of external collaborators you have sponsored access to the Yens for.
```

Comment:
It's not "immediate" -- let's remove that word and add a warning that workgroup changes take up to 24 hours to propagate to the Yen servers.